build/js/intlTelInput.js

```diff
@@ -218,6 +220,15 @@ https://github.com/Bluefieldscom/intl-tel-input.git
                 this.countryList.removeClass("v-hide").addClass("hide");
                 // this is useful in lots of places
                 this.countryListItems = this.countryList.children(".country");
+                // create dropdownContainer markup
+                if (this.options.dropdownContainer) {
+                    this.dropdown = $("<div>", {
+                        "class": "intl-tel-input iti-container"
+                    });
+                    $("<div>", {
+                        "class": "flag-dropdown"
+                    }).appendTo(this.dropdown).append(this.countryList);
```

Comment:
Working on a new method right now - improves load time for inputs both with and without dropdownContainer.